src/Anonymization/Anonymizer/Faker/FakerAddressAnonymizer.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Faker;
+
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AbstractMultipleColumnAnonymizer;
+use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use Faker;
+
+#[AsAnonymizer(
+    name: 'address',
+    pack: 'faker',
+    description: <<<TXT
+    Anonymize a mutlicolumn postal address.
+    Map columns for each part with options ('country', 'locality', 'region', 'postal_code', 'street_address', 'secondary_address').
+    You must set the 'locale' parameter (ex. 'fr_FR', 'en_US', 'de_AT', ...).
+    You can also specify the sample table size using the 'sample_size' option
+    (default is 500). The more samples you have, the less duplicates you will
+    end up with.
+    TXT,
+    requires: [Faker\Factory::class],
+    dependencies: ['fakerphp/faker'],
+)]
+class FakerAddressAnonymizer extends AbstractMultipleColumnAnonymizer
```

Comment:
Not sure we need another specific address anonymizer, is there a way to create a generic mutlicolumn faker anonymizer instead ?

Reply:
Actually the one using Faker is much better, because it supports tens of languages! I think we should probably consider recommend the one using Faker than our own.

Reply:
Close, faker integration is now into its own package.